flintrock/ec2.py

```diff
@@ -57,33 +58,47 @@ def __init__(
             vpc_id: str,
             master_instance: 'boto3.resources.factory.ec2.Instance',
             slave_instances: "List[boto3.resources.factory.ec2.Instance]",
+            use_private_vpc,
```

Comment:
Does this need to be part of the object signature? It seems that whether or not the VPC is private is simply a derived property of `vpc_id`.